src/window-service.js

```diff
@@ -242,15 +243,23 @@
 
                 this.windowTracker.add(newWindow);
 
+                var self = this;
+                var showFunction = function() {
+                    self.$timeout(() => {
+                        newWindow.show();
+                        newWindow.bringToFront();
+                    });
```

Comment:
Timeout is the magic that makes it work. I will change to the lambda, thanks for the tip!